MapboxNavigation/NavigationViewController.swift

```diff
@@ -290,6 +290,8 @@ public class NavigationViewController: NavigationPulleyViewController, RouteMapV
         super.init(coder: aDecoder)
     }
     
+    var updateETAInfoTimer: Timer!
```

Comment:
Can this be an optional instead of an implicitly unwrapped optional? That way the code is explicit about the case where it isn’t set (before `viewWillAppear(_:)`).